templates/partials/hero.html

```diff
@@ -14,5 +14,8 @@ <h1 id="hero-heading" class="text--display">Explore Campaign Finance Data</h1>
           </div>
       </form>
     </div>
+  <script type="application/javascript">
```

Comment:
I actually want these performance marks to be present all the time. I can explain more in a meeting, but essentially these are custom metrics that we could track for various pages. If we keep these marks, it means that performance tracking tools like new relic, or google analytics will automatically pick them up.